db/change_cache.go

```diff
@@ -407,8 +407,18 @@ func (c *changeCache) DocChangedSynchronous(event sgbucket.FeedEvent) {
 				if isDelete {
 					rawBody = nil
 				}
+
+				// Figure out the expiry.  In the case of feed based dcp import, the expiry will be on the
+				// DCP Event.  In the case of feed based migration, the expiry will be in the syncMeta
+				var expiry uint32
+				if event.Expiry > 0 {
+					expiry = event.Expiry
+				} else if syncData.Expiry != nil {
+					expiry = uint32((*syncData.Expiry).Unix())
+				}
+
```

Comment:
^^ I wasn't too sure about this being the right approach, was sort of a "stab" to make the tests pass.

Reply:
Not sure why the expiry is in syncMeta for migration - can't we just use the expiry from the DCP event in both cases?

Reply:
Fixed